packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -355,9 +360,12 @@ export default class BirthdayPicker extends React.Component<Props, State> {
     }
 
     monthsShort(): string[] {
-        const format = new Intl.DateTimeFormat(navigator.language, {
-            month: "short",
-        }).format;
+        const format = new Intl.DateTimeFormat(
+            this.props.locale || navigator.language,
```

Comment:
What do you think about using nullish coalescing here (`??`) instead of logical or to handle the fallback? It might be a little safer if `this.props.locale` is falsy.

Reply:
Updated, thanks Marcy!